static/lssm-v4.user.js

```diff
@@ -39,6 +39,7 @@
 // @match        https://policja.operatorratunkowy.pl/*
 // @match        https://www.operador193.com/*
 // @match        https://www.jogo-operador112.com/*
+// @match        https://www.operador112.com/*
```

Comment:
The script will be generated by another script. Please add the url (and the other stuff) here: https://github.com/LSS-Manager/LSSM-V.4/blob/dev/src/config.ts

Reply:
Na, this will cause problems, as jogo-operador112.com is the "true" pt_PT Url. There seems to be an automatic redirection to the jogo-URL missing. We already asked the game-devs about that situation.

Reply:
May use the police url as alt? And who thought "lets maintain two urls, will be funny" was a good idea? 🙄